### PR TITLE
Exp histgrad refinement

### PR DESCRIPTION
**Pull request by gabrieltorresgamez**
Created because if no PR exists, there is no output

### CODE REVIEW

**Comment by ghost** [comment]
# RUN: HistGradientBoostingClassifier, 2023-05-19 08-11-16

## Hyperparameters

| Path        | Param                         | main   | workspace   |
|-------------|-------------------------------|--------|-------------|
| params.yaml | train_test_split_ratio02.type | person | segment     |

## Metrics

| Path                 | Metric         | main    | workspace   | Change   |
|----------------------|----------------|---------|-------------|----------|
| dvclive/metrics.json | test.epoch.acc | 0.89821 | 0.98525     | 0.08704  |
| dvclive/metrics.json | test.epoch.f1  | 0.89935 | 0.98363     | 0.08428  |

## Plots

### Confusion Matrix

Confusion Matrix

**Comment by gabrieltorresgamez** [comment]
lmao rip florins review hahahahahhaa
